Navtor_intern_task/FleetManagementAppTest/ContainerShipTest.cs

```diff
@@ -0,0 +1,87 @@
+using FleetManagementApp;
+
+namespace FleetManagementAppTest;
+
+public class ContainerShipTest
+{
+
+    
+    [Fact]
+    public void AddContainer_ShouldIncreaseCurrentLoad()
+    {
+        // Arrange
+        var ship = new ContainerShip("IMO9356646", 
+            "TEST", 
+            17, 122, 
+            new Position(new Tuple<double, double>(58.253531, 9.892320)), 
+            8019);
+        
+        var container = new Container("CMAU1234567", "TEST","TEST", 100);
+
+        // Act
+        ship.AddContainer(container);
+
+        // Assert
+        Assert.Equal(100, ship.CurrentLoad);
+    }
+    
+    [Fact]
+    public void RemoveContainer_ShouldDecreaseCurrentLoad()
+    {
+        // Arrange
+        var ship = new ContainerShip("IMO9356646", 
+            "TEST", 
+            17, 122, 
+            new Position(new Tuple<double, double>(58.253531, 9.892320)), 
+            8019);
+        
+        var container = new Container("CMAU1234567", "TEST","TEST", 100);
+        ship.AddContainer(container);
+
+        // Act
+        ship.RemoveContainer(container);
+
+        // Assert
+        Assert.Equal(0, ship.CurrentLoad);
```

Comment:
It rather decreased by removed container, how do you know it was 0?  
Tests are easier to understand if you explicitly introduce `expected` variable. In this case, the expected current load is the initial current load of the ship minus removed container.